reference/configuration/kernel.rst

```diff
@@ -116,15 +126,83 @@ Cache Directory
 
 **type**: ``string`` **default**: ``$this->rootDir/cache/$this->environment``
 
-This returns the path to the cache directory. To change it, override the
-:method:`Symfony\\Component\\HttpKernel\\Kernel::getCacheDir` method. Read
-":ref:`override-cache-dir`" for more information.
+This returns the absolute path of the cache directory of your Symfony project.
+It's calculated automatically based on the current
+:doc:`environment </configuration/environments>`.
+
+This value is exposed via the ``kernel.cache_dir`` configuration parameter and
+the :method:`Symfony\\Component\\HttpKernel\\Kernel::getCacheDir` method. To
+change this setting, override the ``getCacheDir()`` method to return the right
+cache directory.
 
 Log Directory
 ~~~~~~~~~~~~~
 
 **type**: ``string`` **default**: ``$this->rootDir/logs``
 
-This returns the path to the log directory. To change it, override the
-:method:`Symfony\\Component\\HttpKernel\\Kernel::getLogDir` method. Read
-":ref:`override-logs-dir`" for more information.
+This returns the absolute path of the log directory of your Symfony project.
+It's calculated automatically based on the current
+:doc:`environment </configuration/environments>`.
+
+This value is exposed via the ``kernel.log_dir`` configuration parameter and
+the :method:`Symfony\\Component\\HttpKernel\\Kernel::getLogDir` method. To
+change this setting, override the ``getLogDir()`` method to return the right
+log directory.
+
+Container Build Time
+~~~~~~~~~~~~~~~~~~~~
+
+**type**: ``string`` **default**: the result of executing ``time()``
+
+Symfony follows the `reproducible builds`_ philosophy, which ensures that the
+result of compiling the exact same source code doesn't produce different
+results. This helps checking that a given binary or executable code was compiled
+from some trusted source code.
+
+In practice, the compiled :doc:`service container </service_container>` of your
+application will always be the same if you don't change its source code. This is
+exposed via these configuration parameters:
+
+* ``container.build_hash``, a hash of the contents of all your source files;
+* ``container.build_time``, a timestamp of the moment when the container was
+  built (the result of executing PHP's :phpfunction:`time` function);
+* ``container.build_id``, the result of merging the two previous parameters and
+  encoding the result using CRC32.
+
+Since the ``container.build_time`` value will change every time you compile the
+application, the build will not be strictly reproducible. The solution is to use
```

Comment:
maybe add: If you care about this, the solution is [...]

Reply:
That's better! Updated. Thanks.